README.md

```diff
@@ -24,16 +24,20 @@ This project contains the Helm Chart and source code for both the client and ser
 - [Boost](https://www.boost.org)
 - [Amazon AWS C++ SDK](https://github.com/aws/aws-sdk-cpp) (see [below](#installing-the-aws-c-sdk) for instructions on building and installing from source)
 - [yaml-cpp](https://github.com/jbeder/yaml-cpp)
+- [OpenTelemetry C++ SDK](https://github.com/open-telemetry/opentelemetry-cpp) *note: you need to build this with the OTLP providers*
```

Comment:
(nlohman_)json and protobuf should also be listed here I think, since they are not only needed by OpenTelemetry, but also explicitly required in this project's cmake script.

Reply:
The nlohman_json and protobuf libraries are pulled in by the OpenTelemetry libraries.  protobufs are needed for OTLP grpc calls and the I think the OTLP/http calls use the nlohman_json library.

Reply:
Yes, I agree that that is why they are needed. My point is that they should be explicitly listed here (ideally before OpenTelemetry) because they are 1) directly needed for this project to build and 2) needed for building OTLP provider to work, so it helps to tell the user about them.

Reply:
I've updated the README